backend/app/api/endpoints/internal/chat_storage.py

```
@@ -944,6 +948,107 @@ async def get_chat_history(
+    cursor: str
+    next_cursor: Optional[str] = None
+    has_more: bool
+    total_units: int
+
+
+@router.get("/history/{session_id}/subtasks", response_model=SubtaskListResponse)
+async def list_history_subtasks(
+    session_id: str,
+    limit: Optional[int] = Query(None, description="Max summaries to return"),
+    offset: int = Query(0, description="Number of summaries to skip"),
+    db: Session = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)